<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QMainWindow" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="7" column="1">
     <widget class="QPushButton" name="setSalt">
      <property name="text">
       <string>设置盐度</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="feedFish">
      <property name="text">
       <string>喂鱼</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="tempMax"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="setTemp">
      <property name="text">
       <string>设置温度</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_21">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>     盐度:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="saltMax"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="bt_connect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="tempMin"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="saltMin"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="serial">
      <property name="text">
       <string>COM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="area">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="et_send"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sand">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>       区域:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="bt_disconect">
      <property name="text">
       <string>断开连接</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="7">
     <widget class="QTextBrowser" name="message">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="temp">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="salt">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_20">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>     温度:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>上下分别是最大值和最小值</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QPushButton" name="reboot">
      <property name="text">
       <string>重启系统</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
